### DISCUSSION TITLE
Is there a guide on how to use wisky?

### DISCUSSION THREAD

**Discussion by AAfetisov** [Whisky-App/Whisky#20] [Q&A] [answered]
I'm kinda new to all the wine thing and I don't understand how to launch something complex, that requires additional libraries etc, or, for instance, something that requires modified windows registry prior to launch.
thank you

**Comment by amartinez1660** [comment]
+1 on this one. For kicks and giggles I have downloaded Diablo 1 from GoG which I had from before.
I know there's DevilutionX that modernizes all of it but wanted to give it a try through this.

So:
- Downloaded Whisky.zip through the latest of the release tags 
- Got the GamePortingToolkit beta through my own Apple Developer Account
- Followed the 'drop .dmg here' drag and drop windows that appears upon launching whisky (nice take to avoid having to hunt down the folders)
- Created "a bottle", Windows 10. First time using anything Wine related, had heard about bottles before but don't really know what they are or what they do)
- Downloaded the standalone offline GOG Diablo 1 + Hellfire setup executable.
- Tried to run that file. Nothing seems to happen
- The bottle seems to come with default apps like iexplorer, wordpad, etc. tried to run that, nothing happens.
- On the screenshots there seems to be things like DXVK (assuming it's a Vulkan cross DirectX thing). I definitely don't have that.

Kinda lost about what to do or where to go. Definitely missing something as online there's tons of people playing Cyberpunk and whatnot.

In any case, exciting times!

**Reply by zhtianyou** [reply]
+1

**Comment by digitalself** [comment]
Yes, we could really do with any kind of readme or instructions... with Reddit being blacked out right now i can't find any info on how to install games using Whisky and have the exact same issue as amartinez. installers do nothing, can't drag and drop, can't find any other place to install or run anything.

**Comment by IsaacMarovitz** [comment, marked as answer]
I've created a [Wiki](https://github.com/IsaacMarovitz/Whisky/wiki/A-Hitchhiker's-Guide-to-Whisky) :)
